Tests/SentryTests/Networking/SentryReachabilityTests.m

```diff
@@ -37,15 +36,17 @@ @implementation SentryReachabilityTest
 - (void)setUp
 {
     self.reachability = [[SentryReachability alloc] init];
-    // Disable the reachability callbacks, cause we call the callbacks manually.
-    // Otherwise, the reachability callbacks are called during later unrelated tests causing flakes.
-    self.reachability.setReachabilityCallback = NO;
+    // Ignore the actual reachability callbacks, cause we call the callbacks manually.
+    // Otherwise, the actual reachability callbacks are called during later unrelated tests causing
+    // flakes.
+    SentrySetReachabilityIgnoreActualCallback(YES);
```

Comment:
Brilliant, didn't think about that _at all_ 😅 thanks for taking another look at these.

Reply:
Making things testable, although it seems you can't, has always been one of my specialties, and I hate flaky tests.